### PR TITLE
expose worker git details to brigadier-based scripts

### PR DESCRIPTION
**Pull request by krancour**
When working on #1316 I discovered that scripts had no way of getting to commit/ref info.

This PR fixes that.

The API server already includes this information when it creates the k8s secret for an Event. The only issue was that we weren't exposing it neatly via Brigadier.

### CODE REVIEW

File: v2/brigadier/src/workers.ts [hunk 1]
```diff
@@ -25,4 +25,33 @@ export interface Worker {
...
+/**
+ * Represents git-specific Worker details.
+ */
+export interface GitConfig {
+  /**
+   * Specifies the remote repository where, if applicable, the Worker will have
+   * obtained source code from.
+   */
+  cloneURL: string
+	/**
```
Comment (vdice) [inline v2/brigadier/src/workers.ts:43]: nit: spacing inconsistency

File: v2/brigadier/src/workers.ts [hunk 2]
```diff
@@ -25,4 +25,33 @@ export interface Worker {
...
+   * Specifies the exact commit where, if applicable, the Worker will have
+   * obtained source code from.
+   */
+	commit?: string
```
Comment (vdice) [inline v2/brigadier/src/workers.ts:47]: nit: spacing inconsistency

**Review by vdice — APPROVED** [review]

File: v2/brigadier/src/workers.ts (hunk 1 above)
Comment (krancour) [reply v2/brigadier/src/workers.ts:43]: Gah. Thanks!!

File: v2/brigadier/src/workers.ts (hunk 2 above)
Comment (krancour) [reply v2/brigadier/src/workers.ts:47]: Something must be wrong in my editor...

Comment (krancour) [reply v2/brigadier/src/workers.ts:47]: Tabs snuck in somehow even though .ts files are set to use space. 🤷 

Fixing it.

**Pull request merged**